File: 242-Valid-Anagram.cpp
#define IOS   ios_base::sync_with_stdio(false), cin.tie(NULL), cout.tie(0);
class Solution {
public:
    bool isAnagram(string s, string t) {
        IOS
        sort(s.begin(), s.end());
        sort(t.begin(), t.end());

        return s == t;
    }
};

File: 242-Valid-Anagram.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isAnagram = function (s, t) {
    if (s.length !== t.length) return false;
    
    const freqCounter = {};
    for (const char of s) {
        freqCounter[char] = (freqCounter[char] || 0) + 1;
    }

    for (const char of t) {
        if (!freqCounter[char]) return false;
        freqCounter[char] -= 1;
    }
    return true;

    
};